### PR TITLE
WIP New ARM64 ISO with UEFI

### PR DESCRIPTION
**Pull request by klaases**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -292,7 +292,20 @@ minikube_iso: deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/usr/b
 	$(MAKE) BR2_EXTERNAL=../../deploy/iso/minikube-iso minikube_defconfig -C $(BUILD_DIR)/buildroot $(BUILDROOT_OPTIONS)
 	$(MAKE) -C $(BUILD_DIR)/buildroot $(BUILDROOT_OPTIONS) host-python
 	$(MAKE) -C $(BUILD_DIR)/buildroot $(BUILDROOT_OPTIONS)
-	mv $(BUILD_DIR)/buildroot/output/images/rootfs.iso9660 $(BUILD_DIR)/minikube.iso
+	mv $(BUILD_DIR)/buildroot/output/images/boot.iso $(BUILD_DIR)/minikube.iso
+
+minikube_arm_iso: deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/usr/bin/auto-pause # build minikube iso
```
Comment (afbjorklund) [inline Makefile:297]: it is possible to merge these make targets ? Need to build "auto-pause" for the right arch, as well.

File: Makefile [hunk 2]
```diff
@@ -292,7 +292,20 @@ minikube_iso: deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/usr/b
...
+	echo $(ISO_VERSION) > deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/etc/VERSION
+	if [ ! -d $(BUILD_DIR)/buildroot ]; then \
+		mkdir -p $(BUILD_DIR); \
+		git clone --depth=1 --branch=$(BUILDROOT_BRANCH) https://github.com/buildroot/buildroot $(BUILD_DIR)/buildroot; \
+		cp deploy/iso/minikube-iso/go.hash $(BUILD_DIR)/buildroot/package/go/go.hash; \
+	fi;
+	$(MAKE) BR2_EXTERNAL=../../deploy/iso/minikube-iso minikube_arm_defconfig -C $(BUILD_DIR)/buildroot $(BUILDROOT_OPTIONS)
+	$(MAKE) -C $(BUILD_DIR)/buildroot $(BUILDROOT_OPTIONS) host-python
+	$(MAKE) -C $(BUILD_DIR)/buildroot $(BUILDROOT_OPTIONS)
+	mv $(BUILD_DIR)/buildroot/output/images/boot.iso $(BUILD_DIR)/minikube_arm.iso
```
Comment (afbjorklund) [inline Makefile:307]: "arm64" (arm is arm32 / armv7)

File: deploy/iso/minikube-iso/configs/minikube_arm_defconfig [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+# Linux headers same as kernel, a 4.19 series
+BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_4_19=y
+
+# Kernel
+BR2_LINUX_KERNEL=y
+BR2_LINUX_KERNEL_CUSTOM_VERSION=y
+BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.19.202"
+BR2_LINUX_KERNEL_USE_ARCH_DEFAULT_CONFIG=y
+BR2_LINUX_KERNEL_NEEDS_HOST_OPENSSL=y
+BR2_LINUX_KERNEL_INSTALL_TARGET=y
```
Comment (afbjorklund) [inline deploy/iso/minikube-iso/configs/minikube_arm_defconfig:41]: if we are already copying the kernel in the shell script, we don't need to do it with make as well ?

```
cp bzImage root/boot/vmlinuz
```

**Comment by klaases** [comment]
Closing as this will be included in a separate PR.

**Pull request closed without merging**